WalletWasabi/WabiSabi/Models/MultipartyTransaction/MultipartyTransactionParameters.cs

```diff
@@ -35,7 +35,7 @@ public record MultipartyTransactionParameters
 	public ImmutableSortedSet<ScriptType> AllowedOutputTypes { get; init; } = OnlyP2WPKH;
 
 	public int MaxTransactionSize { get; init; } = StandardTransactionPolicy.MaxTransactionSize!.Value;
-	public FeeRate MinRelayTxFee { get; init; } = StandardTransactionPolicy.MinRelayTxFee;
+	public FeeRate MinRelayTxFee { get; init; } = StandardTransactionPolicy.MinRelayTxFee ?? FeeRate.Zero;
```

Comment:
Are we sure about this? @molnard 
I'm not.

Reply:
I think with this you are bypassing a possible error. If the minimum is zero nobody will ever bump into this limitation. Who modified the line above with the exclamation mark, probably he will know this better.

Reply:
The `StandardTransactionPolicy` contains the standard policies for bitcoin transaction which are constant values hardcoded in BitcoinCore. NBitcoin assign a value to all the properties but allows you to do stupid things like assigning `null` to this property in order to bypass the policy check, what  would make this policy a non-standard one. It would have been better to make it immutable and have a `CustomTransactionPolicy` instead but it is what it is.

If we are not going to use this property then we could delete it (this could be the best). In case we want to maintain it and use it to verify the final coinjoin fee rate is above the minimum required for a tx to be broadcasted then we can assume it is not null (because we can see than in the NBitcoin source code) or, check for null and fail in case it is `null`.

Reply:
Then what about this?
```
	public FeeRate MinRelayTxFee { get; init; } = StandardTransactionPolicy.MinRelayTxFee
	?? throw new ArgumentNullExepction(nameof(StandardTransactionPolicy.MinRelayTxFee), "Failed to get non-null value for MinRelayTxFee from StandardTransactionPolicy.")
```

Reply:
Yes, that could work. It is in fact better than removing the property because that is a protocol fork and this PR is for something less important that that.